docs/variants/protectli_vp46xx/building-manual.md

```diff
@@ -18,20 +18,17 @@ VP4630, VP4650 and VP4670.
 > FSP, and ME. Currently, access to them is restricted to the OEM (Protectli) via
 > a private repository.
 
-Since version v1.0.18 VP4630 and VP4650 use different configuration file than
-VP4670. Versions v1.0.17 and older do not support VP4650 and VP4670 at all.
-
```

Comment:
Please revert

Reply:
Added back in 2d72972c1bd0f7ac4b3576c8c9a702d54867efcf